### PR TITLE
[FrameworkBundle][Server Command] add address port number option.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

This allow running server without repeating `127.0.0.1` each time, in the most of case we only change the port number.
- Before

```
app/console server:start 127.0.0.1:8080
```
- After

```
app/console server:start -p 8080
```

### CODE REVIEW

**Comment by dosten** [comment]
:+1:

File: src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php [hunk 1]
```diff
@@ -154,6 +160,11 @@ private function createPhpProcessBuilder(InputInterface $input, OutputInterface
             return;
         }
 
-        return new ProcessBuilder(array($binary, '-S', $input->getArgument('address'), $router));
+        $address = $input->getArgument('address');
+        if (false === strpos($address, ':')) {
+            $address = $address.':'.$input->getOption('port');
+        }
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php]: I think we should rather pass the built address here instead of doing this all again.

Comment (aitboudad) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php]: done

File: src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php [hunk 2]
```diff
@@ -98,12 +99,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $env = $this->getContainer()->getParameter('kernel.environment');
-        $address = $input->getArgument('address');
 
+        $address = $input->getArgument('address');
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php]: Can you revert this change (i.e. moving the blank line down again)?

Comment (aitboudad) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php]: reverted

**Comment by xabbuh** [comment]
:+1:

**Comment by stof** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @aitboudad.
